lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -681,6 +681,12 @@ sub overview {
     );
     my @latest_jobs = $self->schema->resultset('Jobs')->complex_query(%$search_args)->latest_jobs($until);
     ($stash{archs}, $stash{results}, $stash{aggregated}) = $self->prepare_job_results(\@latest_jobs);
+ 
+    # Allow n queries per minute, per user (if logged in)
+    my $lockname = 'webui_overview_rate_limit';
+    if (my $user = $self->current_user) { $lockname .= $user }
+    return $self->render(json => {error => 'Rate limit exceeded'}, status => 400)
+      unless $self->app->minion->lock($lockname, 60, {limit => $self->app->config->{'rate_limits'}->{'overview'}});
```

Comment:
Please replace magic number. I assume 60 means "one minute"? If yes, there is a time date module we can use

Reply:
How did you resolve this? I don't see a change